src/schema/objects/columns.yaml

```diff
@@ -212,6 +356,26 @@ handedness:
 
     For "ambidextrous", use one of these values: `ambidextrous`, `a`, `A`, `AMBIDEXTROUS`,
     `Ambidextrous`.
+hardware_filters:
+  name: hardware_filters
+  display_name: Hardware filters
+  description: |
+    List of hardware (amplifier) filter keys applied.
+    Note that parameters should be defined in the general microephys sidecar .json file.
+    Indicate `n/a` in the absence of hardware filters applied.
+  anyOf:
+    - type: string
+    - type: string
+      enum:
+        - n/a
+height__probes:
+  name: height
+  display_name: Height
+  description: |
+    Physical height of the probe, for example, '5'.
+    This dimension corresponds to the y’axis of the Euler transformation defined by
+    alpha, beta and gamma rotations values below.
+  type: number
```

Comment:
```suggestion
```

I don't understand what this is supposed to be

Reply:
So these are about geometry of the probes. Thus relating to
- #2013 
?